imports/ui/components/partiesList/client/partiesList.tests.js

```diff
@@ -5,55 +5,83 @@ import {
 import 'angular-mocks';
 
 import {
+    ConfigGoogleMap
+} from '../../../configs/googleMap/googleMapConfig';
+
+import {
     chai
 } from 'meteor/practicalmeteor:chai';
 
+import {
+    sinon
+} from 'meteor/practicalmeteor:sinon';
+
 should();
 
 describe('PartiesList', function() {
-    beforeEach(function() {
+    if (!process.env.TESTING)
+        process.env.TESTING = 1;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '{' and instead saw 'process'.](https://www.codacy.com/app/ddspog/MAM-Seed/file/2729684801/issues/source?bid=3340041&fileBranchId=3346427#l23)